packages/airnode-validator/src/ois/ois.ts

```diff
@@ -367,12 +367,19 @@ const ensureEndpointAndApiSpecificationParamsMatch: SuperRefinement<{
   });
 };
 
+export const semverSchema = z.string().refine((value) => {
+  const semver = value.split('.');
+  if (semver.length !== 3) return false;
+
+  return !semver.find((part) => /^\d+$/.test(part) === false);
+}, 'Expected semantic versioning "x.y.z"');
+
```

Comment:
Can't this be done with a simple regex?

Reply:
Not a simple one https://github.com/api3dao/airnode/blob/bb509ca451d9003e9113b2a5f99cad44bdb0b2d5/packages/airnode-validator/templates/1.0/ois.json#L13 (but this one is a bit more flexible)

Anyway, implementing this in JS is much cleaner and maintanable so it is a deliberate choice.

Reply:
It doesn't need to be that complex, we don't use suffixes like `-beta`. It would be something like `^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)$` (haven't tested it though). Anyway, don't really care much, up to you.

Reply:
`^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)$` will not accept `10.20.30` and this is exactly what I want to prevent. Reading the JS implementation is much easier to reason about and get right.

Reply:
Why wouldn't it?
![Screenshot_20220606_091044](https://user-images.githubusercontent.com/1655405/172112957-792d4f56-e702-4edd-9d8e-b00bd26d514f.png)

Reply:
You're right, I missed that you have `\d*` there. I still prefer JS impl though.